config/delta-producer/publication-graph-maintainer/config.json

```
@@ -137,5 +137,29 @@
     "publisherUri": "http://data.lblod.info/services/delta-producer-publication-graph-maintainer-vendor-management",
     "filesGraph": "http://redpencil.data.gift/id/deltas/producer/vendor-management-deltas",
     "account": "http://data.lblod.info/foaf/account/id/0a3d5be8-6f6b-4c4e-9069-9b7c57a35df3"
+  },
+  "worship-services-sensitive": {
+    "errorCreatorUri": "http://lblod.data.gift/services/delta-producer-publication-graph-maintainer-worship-services-sensitive",
+    "exportConfigPath": "/config/worship-services-sensitive/export.json",
+    "filesPath": "/worship-services-sensitive/files",
+    "jobsGraph": "http://mu.semte.ch/graphs/organizations/141d9d6b-54af-4d17-b313-8d1c30bc3f5b/LoketAdmin",
+    "relativeFilePath": "deltas/worship-services-sensitive",
+    "publicationGraph": "http://redpencil.data.gift/id/deltas/producer/loket-worship-services-sensitive",
+    "healingJobOperation": "http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/healingOperation/worship-services-sensitive",
+    "initialPublicationGraphSyncJobOperation": "http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/initialPublicationGraphSyncing/worship-services-sensitive",
+    "queuePollInterval": 3000,
+    "useVirtuosoForExpensiveSelects": true,
+    "skipMuAuthInitialSync": true,
+    "skipMuAuthHealing": true,
+    "useFileDiff": true,
+    "healingInitialBatchSizeInsert": 1000,
+    "updatePublicationGraphSleep": 200,
+    "serveDeltaFiles": true,
+    "deltaInterval": 10000,
+    "deltaPath": "/worship-services-sensitive/delta",
+    "loginPath": "/worship-services-sensitive/login",
+    "publisherUri": "http://data.lblod.info/services/delta-producer-publication-graph-maintainer-ws-sensitive",
```

I was thinking of renaming this line to:
```suggestion
    "publisherUri": "http://data.lblod.info/services/delta-producer-publication-graph-maintainer-worship-services-sensitive",
```
to match the format of the other keys.

---

Good catch I've missed that one